src/aics-image-viewer/components/ControlPanel/index.tsx

```diff
@@ -157,4 +173,4 @@ function ControlPanel(props: ControlPanelProps): React.ReactElement {
   );
 }
 
-export default React.memo(ControlPanel);
+export default React.memo(connectToViewerState(ControlPanel, ["resetToDefaultViewerState"]));
```

Comment:
`connectToViewerState` already memoizes the passed component:


```suggestion
export default connectToViewerState(ControlPanel, ["resetToDefaultViewerState"]);
```